### PR TITLE
Update volunteer log sync

### LINKED ISSUE #204: Syncing volunteer logs should support logs created for other users

### PR DESCRIPTION
**Pull request by eliascodes**
Fixes #204  

### Changes
* Allow `VOLUNTEER_ADMIN` users to sync logs corresponding to sibling users
* Fix remaining 'ORG_ADMIN's
* Remove explicit declarations of `failAction`, and keep this all in `api/v1/index.ts`, adding the default `failAction` if routes don't already have it defined.

### CODE REVIEW

File: src/api/v1/community_businesses/volunteer_logs.ts [hunk 1]
```diff
@@ -294,18 +295,30 @@ const routes: Hapi.ServerRoute[] = [
...
       const {
         server: { app: { knex } },
-        auth: { credentials: { user } },
+        auth: { credentials: { user, scope } },
         pre: { communityBusiness },
         payload,
       } = request;
 
+      // TODO:
+      // // Edge case: We don't check each target userId (if it exists)
```
Comment (astroash) [inline src/api/v1/community_businesses/volunteer_logs.ts]: 🤔 we shouldn't allow users to create logs for non volunteers.

File: src/api/v1/community_businesses/volunteer_logs.ts [hunk 2]
```diff
@@ -294,18 +295,30 @@ const routes: Hapi.ServerRoute[] = [
...
+      // //            corresponds to a VOLUNTEER(_ADMIN)
+
+      if (
```
Comment (astroash) [inline src/api/v1/community_businesses/volunteer_logs.ts]: Can you comment what this conditional is doing.

File: docs/README.md [hunk 3]
```diff
@@ -2,13 +2,14 @@
...
-3. [Architecture](./architecture.md)
-4. [Configuration](./configuration.md)
-5. [Database](./database.md)
-6. [Error Handling](./error_handling.md)
-7. [Getting started](./getting_started.md)
-8. [Permissions](./permissions.md)
-9. [Stack](./stack.md)
-10. [Deployment](./deployment.md)
-11. [Survey Benchmarks](./survey_benchmarks.md)
+1. [API Migration](./api_migration.md)
```
Comment (astroash) [inline docs/README.md]: I know it won't show, but this numbering is making me sad...

File: docs/database_backups.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# 🗃 Database Backups
```
Comment (astroash) [inline docs/database_backups.md]: Did you credit the author? 🙃

File: docs/README.md (hunk 3 above)
Comment (eliascodes) [reply docs/README.md]: I know, me too, but I'm made happy by the thought I won't ever have to re-number any of them again.

**Pull request merged**